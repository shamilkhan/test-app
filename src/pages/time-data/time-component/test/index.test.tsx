import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import TimeComponent from "../index";

describe("<TimeComponent />", () => {
  it("TimeStamp rendered in document", () => {
    const TIME_STAMP = 12345667;
    const { getByText } = render(
      <TimeComponent isConnected timestamp={TIME_STAMP} />
    );
    const node = getByText(TIME_STAMP.toString());
    expect(node).toBeInTheDocument();
  });
  it("SnapShot", () => {
    const tree = renderer
      .create(<TimeComponent isConnected timestamp={111111111} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
