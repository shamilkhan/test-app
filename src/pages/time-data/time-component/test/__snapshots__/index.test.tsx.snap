// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeComponent /> SnapShot 1`] = `
<div
  className="time-data__value"
>
  <div
    className="time-data__icon"
  >
    ðŸ•”
  </div>
  <div
    className="time-data__timestamp"
  >
    111111111
  </div>
</div>
`;

exports[`<TimeComponent /> Snapshot 1`] = `
<div
  className="time-data__value"
>
  <div
    className="time-data__icon"
  >
    ðŸ•”
  </div>
  <div
    className="time-data__timestamp"
  >
    12312
  </div>
</div>
`;
